stats/Record.md

```diff
@@ -104,3 +110,11 @@ if (spanContext.getTraceOptions().isSampled()) {
   measurementMap.record(tagContext);
 }
 ```
+
+## Flush
```

Comment:
This semantic should be added for trace as well. You can just open an issue for that or do the change, up to you.

Reply:
Are you suggesting that the library should provide a top-level opencensus.Flush() that flushes everything? 

If so, I agree. But I think this then needs to be documented somewhere else.